packages/plugin-credentials/src/credentials/common.js

```diff
@@ -368,16 +391,16 @@ export default {
       }
 
       attrs.userTokens = attrs.userTokens || [];
-
-      if (attrs.apiToken) {
-        attrs.userTokens.push(attrs.apiToken);
-        Reflect.deleteProperty(attrs, `apiToken`);
-      }
-
-      if (attrs.kmsToken) {
-        attrs.userTokens.push(attrs.kmsToken);
-        Reflect.deleteProperty(attrs, `kmsToken`);
-      }
+      // // TODO: is apiToken and kmsToken needed? should already be in userTokens?
```

Comment:
This is for compatibility with the old storage formats. we need to make sure we don't log people out when they get the new sdk version